src/google/google-calendars/googlecalendarsyncadaptor.h

```diff
@@ -77,7 +77,13 @@ private Q_SLOTS:
     void upsyncFinishedHandler();
 
 private:
-    QMap<int, QMap<QString, QPair<QString, QString> > > m_serverCalendarIdToSummaryAndColor;
+    struct CalendarInfo {
+        QString summary;
+        QString description;
+        QString color;
+        QString etag;
```

Comment:
I have now removed it again, as we probably won't use it in the future.  If we do, we can add it then.